.github/workflows/ci.yml

```
@@ -20,10 +20,11 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install black flake8 mypy
+        python -m pip install -r requirements.txt
+        python -m pip install black flake8 mypy
     - name: Format
```

```suggestion
    - name: black
```
Might as well do this for other checks and not just flake8